internal/pkg/ccloudv2/stream-designer.go

```diff
@@ -55,17 +55,23 @@ func (c *Client) executeListPipelines(envId, clusterId, pageToken string) (strea
 func (c *Client) CreatePipeline(envId, clusterId, name, description, sourceCode string, secretMappings *map[string]string, ksqlId, srClusterId string) (streamdesignerv1.SdV1Pipeline, error) {
```

Comment:
It might be a better idea to construct the pipeline object outside of this function and pass it in (instead of 8 arguments!)